app/scripts/lib/sms-client.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A client to send SMS messages
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const xhr = require('lib/xhr');
+
+  const SMSClient = {
+    /**
+     * Send an SMS message
+     *
+     * @param {String} phoneNumber
+     * @param {String} email
+     * @returns {Promise}
+     */
+    send (phoneNumber, email) {
```

Comment:
This might end up moving into the account, so that the request to send an SMS is sent with a sessionToken which can be used to validate a user/rate limit SMS send requests. Or, maybe the send function can stay here and instead of an email, a `sessionToken` is sent. I think I like the former more.